app/assets/javascripts/application.js.erb

```diff
@@ -95,13 +95,10 @@ jQuery(document).ready(function ($) {
   });
 });
 
-function checkAll (id, checked) {
-  var els = Element.descendants(id);
-  for (var i = 0; i < els.length; i++) {
-    if (els[i].disabled === false) {
-      els[i].checked = checked;
-    }
-  }
+function checkAll(selector, checked) {
+  jQuery('#' + selector + ' input:checkbox').each(function() {
+    this.checked = checked;
+  });
```

Comment:
You'll have to exclude diabled checkboxes because it is otherwise possible to change the value of disabled fields:

![image](https://cloud.githubusercontent.com/assets/617519/9490758/5e0af7d2-4bea-11e5-9fef-1928e741d655.png)